source/content/php-versions.md

```diff
@@ -28,8 +28,8 @@ Changes made to the `pantheon.yml` file on a branch **are not** detected when cr
 
 | Version                                      | Recommended |  Status |
 | --------------------------------------------:|:-----------:| ------- |
-| 8.1 | ➖ | Available <Popover title="New Relic" content="New Relic is not supported in PHP 8.1." /> |
-| [8.0](https://v80-php-info.pantheonsite.io/) | ✅          | Active  |
+| [8.1](https://v81-php-info.pantheonsite.io/)| ➖ | Available <Popover title="Compatibility Note" content="WordPress is not fully compatible with PHP 8.1. New Relic is not supported in PHP 8.1." /> |
+| [8.0](https://v80-php-info.pantheonsite.io/) | ✅          | Active <Popover title="Compatibility Note" content="WordPress is not fully compatible with PHP 8.0." /> |
```

Comment:
I believe the content here should direct the reader to the Compatibility Considerations section so we're not repeating ourselves.

Reply:
Thank you @CdrMarks. It's a good idea to direct users to that section. I'll open a new PR to add that change. Thanks a miilion